### PR TITLE
Change github corner

### PR DESCRIPTION
**Pull request by cedced19**
We still use an image although we can do this with css and svg.
See this link: [http://tholman.com/github-corners/](http://tholman.com/github-corners/)
Before:
imgrevs
After:
imgres

### CODE REVIEW

File: css/app.less [hunk 1]
```diff
@@ -225,6 +225,22 @@ footer {
 	animation-delay: -0.16s;
 }
 
+.github-corner:hover .octo-arm {
+	animation:octocat-wave 560ms ease-in-out
+}
+
+@-webkit-keyframes octocat-wave {
```
Comment (prometheansacrifice) [inline css/app.less]: @cedced19 Only webkit prefix?

Comment (cedced19) [inline css/app.less]: There are only webkit prefix in other animation too

Comment (prometheansacrifice) [inline css/app.less]: Yes. But why?

Comment (cedced19) [inline css/app.less]: I don't know, maybe we can add a autoprefixer task after added my pull request.

**Comment by contolini** [comment]
I dig the new corner. :+1:

File: css/app.less (hunk 1 above)
Comment (fahad19) [inline css/app.less]: there is autoprefixer in the Gruntfile: https://github.com/bower/search/blob/gh-pages/Gruntfile.js#L20

Comment (cedced19) [inline css/app.less]: Ok, I can remove prefix then.

**Comment by zzarcon** [comment]
:heart_eyes_cat: :+1:

**Pull request merged**